.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+repos:
+- repo: https://github.com/antonbabenko/pre-commit-terraform
+  rev: v1.68.1
```

Comment:
Should we commit this with `--freeze` so as to reference a hash rather than a tag which can be changed easily? We're running this in an environment which potentially contains secrets that can access infrastructure.

Reply:
I'll go ahead and do that, but in any case it shouldn't be a problem either way, since pre-commit hooks are solely ran locally and so should have no way to access the Actions Secrets where the credentials are stored. On a similar note, do you think it would be sufficient to pin the dflook action to a specific hash instead of just the `v1` tag?

Reply:
Given your local machines currently hold these keys and likely hold keys with more privilege (for example, any other cloud or internal networks you might working on), I would consider running arbitrary code on them more dangerous than GitHub Actions (which should have highly scoped credentials) as well as providing that same risk to others working in this repository.

I agree with you that fixing it to a hash for `dflook` is the right thing to do, should be enough to stop low effort attacks.